src/Razor/Microsoft.AspNetCore.Razor.Tools/src/Application.cs

```diff
@@ -39,6 +39,7 @@ internal class Application : CommandLineApplication
             Commands.Add(new ShutdownCommand(this));
             Commands.Add(new DiscoverCommand(this));
             Commands.Add(new GenerateCommand(this));
+            Commands.Add(new BrotliCompressCommand(this));
```

Comment:
Does this ties compression to the Razor compiler?

Reply:
To the rzc tool, but it does not remote calls to the server process. It doesn't seem problematic as yet, but we can come back to this if there are side-effects to it.